src/output.cpp

```diff
@@ -0,0 +1,736 @@
+  } else {
+    std::cout << "  Energy Group = " << p->g_ << "\n";
+  }
+  std::cout << "  Delayed Group = " << p->delayed_group_ << "\n";
+
+  std::cout << "\n";
+}
+
+//==============================================================================
+
+void print_plot()
+{
+  header("PLOTTING SUMMARY", 5);
+  if (settings::verbosity < 5) return;
+
+  for (auto pl : model::plots) {
+    // Plot id
+    std::cout << "Plot ID: " << pl.id_ << "\n";
+    // Plot filename
+    std::cout << "Plot file: " << pl.path_plot_ << "\n";
+    // Plot level
+    std::cout << "Universe depth: " << pl.level_ << "\n";
+
+    // Plot type
+    if (PlotType::slice == pl.type_) {
+      std::cout << "Plot Type: Slice" << "\n";
+    } else if (PlotType::voxel == pl.type_) {
+      std::cout << "Plot Type: Voxel" << "\n";
+    }
+
+    // Plot parameters
+    std::cout << "Origin: " << pl.origin_[0] << " "
+              << pl.origin_[1] << " "
+              << pl.origin_[2] << "\n";
+
+    if (PlotType::slice == pl.type_) {
+      std::cout << std::setprecision(4)
+                << "Width: "
+                << pl.width_[0] << " "
+                << pl.width_[1] << "\n";
+    } else if (PlotType::voxel == pl.type_) {
+      std::cout << std::setprecision(4)
+                << "Width: "
+                << pl.width_[0] << " "
+                << pl.width_[1] << " "
+                << pl.width_[2] << "\n";
+    }
+
+    if (PlotColorBy::cells == pl.color_by_) {
+      std::cout << "Coloring: Cells" << "\n";
+    } else if (PlotColorBy::mats == pl.color_by_) {
+      std::cout << "Coloring: Materials" << "\n";
+    }
+
+    if (PlotType::slice == pl.type_) {
+      switch(pl.basis_) {
+      case PlotBasis::xy:
+        std::cout <<  "Basis: XY" << "\n";
+        break;
+      case PlotBasis::xz:
+        std::cout <<  "Basis: XZ" << "\n";
+        break;
+      case PlotBasis::yz:
+        std::cout <<  "Basis: YZ" << "\n";
+        break;
+      }
+      std::cout << "Pixels: " << pl.pixels_[0] << " "
+                << pl.pixels_[1] << " " << "\n";
+    } else if (PlotType::voxel == pl.type_) {
+      std::cout << "Voxels: " << pl.pixels_[0] << " "
+                << pl.pixels_[1] << " "
+                << pl.pixels_[2] << "\n";
+    }
+
+    std::cout << "\n";
+
+  }
+}
+
+//==============================================================================
+
+void
+print_overlap_check()
+{
+  #ifdef OPENMC_MPI
+    std::vector<int64_t> temp(model::overlap_check_count);
+    MPI_Reduce(temp.data(), model::overlap_check_count.data(),
+               model::overlap_check_count.size(), MPI_INT64_T,
+               MPI_SUM, 0, mpi::intracomm);
+  #endif
+
+  if (mpi::master) {
+    header("cell overlap check summary", 1);
+    std::cout << " Cell ID      No. Overlap Checks\n";
+
+    std::vector<int32_t> sparse_cell_ids;
+    for (int i = 0; i < model::cells.size(); i++) {
+      std::cout << " " << std::setw(8) << model::cells[i]->id_ << std::setw(17)
```

Comment:
Optional: Looks like you're setting standard widths for the cell_id and check_count_outputs here.  Just wanted to call out that there is no space in the `cout` stream between the cell `id_` and the `overlap_check_count`.

If this output is being fed into another system where precise widths are necessary, this isn't a problem.  However if the `overlap_check_count` winds up being a width of 17, the `id` and the `overlap_check_count` will bump up against one another without a separator.

Would recommend adding a space between `id_` and `overlap_check_count`, similar to elsewhere in the codebase.